wiki/Tournaments/ATS/2018/en.md

```diff
@@ -0,0 +1,170 @@
+---
+tags:
+- ATS 2018
+- ATS2018
+---
+Asian Taiko Showdown 2018
+=============================
+
+![Asian Taiko Showdown 2018](banner.png)
+
+> Unite the Diversity between us!
+
+**ATS 2018**, short for **Asian Taiko Showdown 2018**, is a double elimination one-on-one osu!taiko tournament hosted by [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002). Previously, ATS was known as [Indonesian Taiko Showdown (IDTS)](https://osu.ppy.sh/community/forums/topics/576070). This is the 2nd installment of the tournament and extension from the Indonesia region to Asia.
```

Comment:
**ATS 2018**, short for **Asian Taiko Showdown 2018**, -> The **Asian Taiko Showdown 2018** (***ATS 2018***)